R/importHumann.R

```diff
@@ -3,19 +3,21 @@
 #' @param file a single \code{character} value defining the file
 #' path of the HUMAnN file. The file must be in merged HUMAnN format.
 #'
-#' @param colData a DataFrame-like object that includes sample names in
+#' @param col.data a DataFrame-like object that includes sample names in
 #' rownames, or a single \code{character} value defining the file
 #' path of the sample metadata file. The file must be in \code{tsv} format
-#' (default: \code{colData = NULL}).
+#' (default: \code{col.data = NULL}).
+#' 
+#' @param colData Deprecated. Use \code{col.data} instead.
 #' 
 #' @param ... additional arguments:
 #' \itemize{
 #'   \item \code{assay.type}: A single character value for naming 
 #'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{assay}} 
 #'   (default: \code{assay.type = "counts"})
-#'   \item \code{removeTaxaPrefixes}: \code{TRUE} or \code{FALSE}: Should
+#'   \item \code{remove.taxa.prefix}: \code{TRUE} or \code{FALSE}: Should
 #'     taxonomic prefixes be removed? (default:
-#'     \code{removeTaxaPrefixes = FALSE})
+#'     \code{remove.taxa.prefix = FALSE})
```

Comment:
The name could be also taxa.prefix.rm. Not sure if that is any better (the name is quite long)

Reply:
or `rm.prefix` - the purpose might be clear from the context, and rm is widely used abbreviation. Although remove.taxa.prefix is more clear.

Reply:
i would do rm.* rather than *.rm (think about rm.na for instance, let's stay consistent)

Reply:
so rm.*?

Reply:
I did not get the point of rm.*

na.rm is widely used in R, so based on that, the justification is for prefix.rm

Reply:
ACK.